pop-api/integration-tests/src/fungibles/mod.rs

```diff
@@ -364,8 +382,14 @@ fn instantiate_and_create_fungible_works() {
 			Err(Module { index: 52, error: 5 })
 		);
 		// Successfully create an asset when instantiating the contract.
-		assert_ok!(instantiate_and_create_fungible(contract, ASSET_ID, 1));
+		let instantiator =
+			instantiate_and_create_fungible(contract, ASSET_ID, 1).expect("Should work");
 		assert!(Assets::asset_exists(ASSET_ID));
+		// Successfully emit an event from instantiating a new fungible.
```

Comment:
Can we change all the event checks to the following: 
```suggestion
		// Successfully emit event.
```

One reason is simplification and the other reason is the use of the word "asset". The latter is something we don't want to anymore, always refer to "token".